l2l/optimizers/multievolution/optimizer.py

```diff
@@ -0,0 +1,239 @@
+import logging
+import random
+
+from collections import namedtuple
+
+from deap import base, creator, tools
+from deap.tools import HallOfFame
+
+from l2l import dict_to_list, list_to_dict
+from l2l.optimizers.optimizer import Optimizer
+
+logger = logging.getLogger("l2l-ga")
+
+MultiGeneticAlgorithmParameters = namedtuple('GeneticAlgorithmParameters',
+                                        ['seed', 'pop_size', 'cx_prob', 'mut_prob', 'n_iteration', 'ind_prob', 'tourn_size', 'mate_par',
+                                         'mut_par', 'n_inner_params'])
+MultiGeneticAlgorithmParameters.__doc__ = """
```

Comment:
The code needs a better documentation. Especially the intend/goal of the Multi-optimization should be mentioned to separate it from the normal GA and GD.